build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -1589,7 +2201,18 @@ diff --git a/components/subresource_filter/content/browser/ruleset_service.cc b/
 diff --git a/components/subresource_filter/content/browser/ruleset_service.h b/components/subresource_filter/content/browser/ruleset_service.h
 --- a/components/subresource_filter/content/browser/ruleset_service.h
 +++ b/components/subresource_filter/content/browser/ruleset_service.h
-@@ -182,7 +182,7 @@ class RulesetService : public base::SupportsWeakPtr<RulesetService> {
+@@ -171,6 +171,10 @@ class RulesetService : public base::SupportsWeakPtr<RulesetService> {
+     publisher_->SetRulesetPublishedCallbackForTesting(std::move(callback));
+   }
+ 
++  void SetRulesetPublishedCallback(base::OnceClosure callback) {
++    publisher_->SetRulesetPublishedCallbackForTesting(std::move(callback));
```

Comment:
I meant to completely avoid calling this method and have our own, doing the identical job.

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/51b8c9b04f86cb1cb56e30abc152bfd539780bf9